### PR TITLE
add hover function + disclaimer css

### PR DESCRIPTION
**Pull request by mirnawong1**
this pr adds the hover functionality and disclaimer css to the existing faq component. this is because our detailsToggle component has a hover functionality and i think we should have consistent behavior for users.

this PR also removes the underline under the 'hover to view' string.

### CODE REVIEW

**Comment by nghi-ly** [comment]
hey @mirnawong1 ! i really like this and appreciate the consistency across the faqs. 

for a little more polish 💅 , is it possible to have the text "Hover to view" _not_ be underlined since that's not actually part of the faq content (it's just a helper note) ?

**Comment by mirnawong1** [comment]
thanks @nghi-ly and that's a good suggestion! I'll look into to removing the underline so the distinction is clearer 🙌

**Comment by mirnawong1** [comment]
update: i've removed the underline from the 'hover to view' for both the faq and detailsToggle components.

File: website/src/components/detailsToggle/index.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ useEffect(() => {
         onMouseLeave={handleMouseLeave}
       >
         <span className={`${styles.toggle} ${isOn ? null : styles.toggleUpsideDown}`}></span>&nbsp;
-        <span>{alt_header}</span>
+        <span className={styles.headerText}>{alt_header}</span>
```
Comment (mirnawong1) [inline website/src/components/detailsToggle/index.js:43]: adding span to separate alt_header and .disclaimer CSS and assign underline class to alt_header only.

**Review by JKarlavige — APPROVED** [review]
all lgtm! verified faqs opening on hover, while still being able to click to open the faqs. Great work @mirnawong1! :shipit:

**Comment by mirnawong1** [comment]
thanks so much for the review @JKarlavige!

**Pull request merged**